app/models/talk.rb

```diff
@@ -182,6 +182,7 @@ def formatted_kind
   scope :for_topic, ->(topic_slug) { joins(:topics).where(topics: {slug: topic_slug}) }
   scope :for_speaker, ->(speaker_slug) { joins(:speakers).where(speakers: {slug: speaker_slug}) }
   scope :for_event, ->(event_slug) { joins(:event).where(events: {slug: event_slug}) }
+  scope :watchable, -> { where(video_provider: [:youtube, :mp4, :vimeo, :parent]) }
```

Comment:
Something to be aware of: the provider of a `parent` talk can still be `scheduled`, `not_published` or `not_recorded` which is technically not watchable. 

Ideally it would be the union of:
```ruby
Talk.where(video_provider: [:youtube, :mp4, :vimeo])
Talk.joins(:parent_talk).where(parent_talks_talks: { video_provider: [:youtube, :mp4, :vimeo] })
```

But I guess for the sake of simplicity let's just do:
```suggestion
  scope :watchable, -> { where(video_provider: [:youtube, :mp4, :vimeo]) }
```

That way you only get the ones which are really "watchable", which also includes all child talks.